addons/sys_intercom/fnc_seatSwitched.sqf

```diff
@@ -54,11 +56,9 @@ if (GVAR(broadcastKey)) then {
         if (_connectionStatus > INTERCOM_DISCONNECTED) then {
             [_vehicle, _unit, _forEachIndex] call FUNC(setStationUnit);
         };
-        [_vehicle, _unit] call FUNC(updateVehicleInfoText);
```

Comment:
Here it was called  in order to remove as soon as possible the text in green when using the intercom. Is it still working the same way?

Reply:
From what I can see, yes. This loop over intercoms is really fast, because there are never a lot of intercoms, so calling it once for all of them at the end is actually less time spent in UI work.

Reply:
It is indeed faster, just tested.

Only not-instant part is switching certain seats due to 1.1s intercom PFH, but those eats cannot be caught by anything else as there are no EHs that work reliably or support all possibilities.

Why is that EH 1.1s btw? 1s would seem more "standard"? :thinking:

Reply:
see above